driver-core/src/main/java/com/datastax/driver/core/ExecutionInfo.java

```diff
@@ -119,4 +127,8 @@ public ConsistencyLevel getAchievedConsistencyLevel() {
     public QueryTrace getQueryTrace() {
         return trace;
     }
+
+
+
+    public PagingState getPagingState(Statement statement) { return PagingState.wrap(usedPagingState, statement); }
```

Comment:
`usedPagingState` can be `null` if we're on the last page. We should probably return `null` in that case.